pkg/csi/service/wcp/controller.go

```diff
@@ -62,6 +62,9 @@ var (
 
 var getCandidateDatastores = cnsvsphere.GetCandidateDatastoresInCluster
 
+// contains list of clusterComputeResourceMoIds on which supervisor cluster is deployed
```

Comment:
nit: `contains` -> `Contains` and end with fullstop.

Reply:
done.